src/main/java/by/itech/lab/supplier/repository/CarRepository.java

```diff
@@ -0,0 +1,24 @@
+package by.itech.lab.supplier.repository;
+
+import by.itech.lab.supplier.domain.Car;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+public interface CarRepository extends JpaRepository<Car, Long> {
+
+    @Modifying
+    @Query("update Car set deletedAt = current_timestamp where id = :id")
+    void deleteById(@Param("id") Long id);
+
+    @Modifying
+    @Query("update Car set currentCapacity = currentCapacity - :capacity  where id = :id")
+    int decreaseCurrentCapacity (@Param("capacity") double capacity, @Param("id") Long id);
```

Comment:
is that method used somewhere?

Reply:
also, what about usage of only one method with {{currentCapacity + :capacity}} where as capacity put values above zero to increase and below zero to decrease?
for example:
300 + (-10) = 290
300 + (10) = 310

Reply:
Can I add TODO with description where to call method in future, when the services that call this method will be developed?

Reply:
no, in most of the cases there is a really low chance that you will work on that part and most possible that someone who will work on that task will implement his own solution, as he doesn't know that this part is already implemented or he will see the implementation in another way